docs/root/intro/arch_overview/statistics.rst

```diff
@@ -4,23 +4,39 @@ Statistics
 ==========
 
 One of the primary goals of Envoy is to make the network understandable. Envoy emits a large number
-of statistics depending on how it is configured. Generally the statistics fall into two categories:
+of statistics depending on how it is configured. Generally the statistics fall into three categories:
 
 * **Downstream**: Downstream statistics relate to incoming connections/requests. They are emitted by
   listeners, the HTTP connection manager, the TCP proxy filter, etc.
 * **Upstream**: Upstream statistics relate to outgoing connections/requests. They are emitted by
   connection pools, the router filter, the TCP proxy filter, etc.
+* **Server**: Server statistics describe how the Envoy server instance is working. Statistics like
+  server uptime or amount of allocated memory are categorized here.
 
 A single proxy scenario typically involves both downstream and upstream statistics. The two types
 can be used to get a detailed picture of that particular network hop. Statistics from the entire
 mesh give a very detailed picture of each hop and overall network health. The statistics emitted are
 documented in detail in the operations guide.
 
-Envoy uses statsd as the statistics output format, though plugging in a different statistics sink
-would not be difficult. Both TCP and UDP statsd is supported. Internally, counters and gauges are
-batched and periodically flushed to improve performance. Histograms are written as they are
-received. Note: what were previously referred to as timers have become histograms as the only
-difference between the two representations was the units.
+In the v1 API, Envoy only supports statsd as the statistics output format. Both TCP and UDP statsd
+are supported. As of the v2 API, Envoy has the ability to support custom, pluggable sinks. A few
```

Comment:
Can we link https://github.com/envoyproxy/data-plane-api/pull/544/files#diff-576397c23369a999e5c1403324db288fL17 (`StatsSink`) when referring to the v2 API so that folks know what is supported?

Reply:
+1. At this phrase? e63db2d